src/endpoints/accounts.ts

```diff
@@ -0,0 +1,16 @@
+import { client } from 'services/client';
+import { BaseResponse } from 'types';
+import { AccountResponse } from './account';
+
+export interface AccountsResponse extends BaseResponse {
+    data: AccountResponse['data'][];
+    links: {
+        self: string;
+    };
+}
+
+export const accountsEndpoint = {
+    read: () => {
+        return client<AccountsResponse>('accounts');
+    },
+};
```

Comment:
This one is kinda weird and not sure about.... we have the concept of singular and plural endpoints. So I broke them into two files and just had the plural one reference the singular one's data.

Reply:
Curious if endpoints that return and array of items could be appended with `-list` (e.g., `/accounts` becomes `/account-list`). If it is determined that the endpoint should not change, applying the aforementioned convention to file names and assets of plural endpoints may be worth consideration (e.g., _src/endpoints/accounts.ts_ becomes _src/endpoints/account-list.ts_).